projects/1.16/assets/relics-of-gaming/rogblades/lang/zh_cn.json

```
@@ -0,0 +1,42 @@
+{
+   "itemGroup.rog_mod":"拔刀剑：游戏遗物",
+
+  "rogblades.name": "拔刀剑：游戏遗物",
+  "rogblades.desc": "来自游戏中的武器。",
+
+  "item.slashblade.ex.beastlord": "百兽剑王",
+  "item.slashblade.ex.beastlord.desc": "游戏《尼尔：机械纪元》中的武器。",
+  "item.slashblade.ex.beastlord.material": "百兽剑王拵",
+
+  "item.slashblade.ex.virtuous": "白之契约",
+  "item.slashblade.ex.virtuous.desc": "游戏《尼尔：机械纪元》中的武器。",
+  "item.slashblade.ex.virtuous.material": "白之契约拵",
+
+  "item.slashblade.ex.beastlord2": "百兽剑王「觉醒」",

```

```suggestion
  "item.slashblade.ex.beastlord2": "觉醒的百兽剑王",
```
中文/日语的`「XX」`在英语文件内均对应`-XX-`，不要加戏
译名格式暂时按照本体锈刀去处理，有更好的另说